data/transactions/logic/assembler.go

```diff
@@ -2584,48 +2684,17 @@ func (ops *OpStream) prependCBlocks() []byte {
 	return out
 }
 
-func (ops *OpStream) error(problem interface{}) error {
-	return ops.lineError(ops.sourceLine, problem)
-}
-
-func (ops *OpStream) lineError(line int, problem interface{}) error {
-	var err lineError
-	switch p := problem.(type) {
-	case string:
-		err = lineError{Line: line, Err: errors.New(p)}
-	case error:
-		err = lineError{Line: line, Err: p}
-	default:
-		err = lineError{Line: line, Err: fmt.Errorf("%#v", p)}
-	}
-	ops.Errors = append(ops.Errors, err)
-	return err
-}
-
-func (ops *OpStream) errorf(format string, a ...interface{}) error {
-	return ops.error(fmt.Errorf(format, a...))
-}
-
-func (ops *OpStream) lineErrorf(line int, format string, a ...interface{}) error {
-	return ops.lineError(line, fmt.Errorf(format, a...))
+// record puts an error onto a list for reporting later. The hope is that the
+// assembler can keep going after an error that is "swallowed" this way, so that
+// more than one TEAL error can be reported in one pass.  By explicitly
```

Comment:
```suggestion
// more than one TEAL error can be reported in one pass.  By implicitly
```

Reply:
if `f()` may return an error, I would consider a line like:
```
f()
```
to be _implicitly_ swallowing the error, and the linter should complain, and we had to turn it off because we used `f()` this way a lot.  But
```
ops.record(f())
```
Seems to be _explicitly_ taking the error and doing "something" with it, so the linter doesn't complain.